tests/expectations/cat-hs-x-mr-pw-idxs.py

```diff
@@ -0,0 +1,10 @@
+[
+    [(1, 2, 3, 4), (2, 3), (), (), (2,)],
+    [(), (), (), (), (3,)],
+    [(1, 2, 3, 4), (2, 3), (), (), (3,)],
+    [(), (), (), (), ()],
+    [(), (0,), (0,), (0,), (0,)],
+    [(), (), (), (0, 1, 4), ()],
+    [(), (), (), (), ()],
+    [(), (0,), (0, 1), (0, 1, 4), (0,)],
+]
```

Comment:
out of curiosity, saving numpy array on disk using py file instead of binary npy files is just for having readable expectations? cause i always used .npy file to preserve numpy arrays on disk and also to reduce size ... but i don't know if is this the case to use the binary files rather than normal py files

Reply:
oh, maybe they're not all numpy arrays but standard python structures